spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -139,21 +155,14 @@ function setup() {
 }
 ```
 
-Once the `setup` function has been called, channels can be created through the IBC routing module between instances of the ibc account module on separate chains.
-
-An administrator (with the permissions to create connections & channels on the host state machine) is responsible for setting up connections to other state machines & creating channels
-to other instances of this module (or another module supporting this interface) on other chains. This specification defines packet handling semantics only, and defines them in such a fashion
-that the module itself doesn't need to worry about what connections or channels might or might not exist at any point in time.
-
-### Routing module callbacks
+Once the `setup` function has been called, channels can be created via the IBC routing module.
 
 ### Channel lifecycle management
 
-Both machines `A` and `B` accept new channels from any module on another machine, if and only if:
+An interchain account module will accept new channels from any module on another machine, if and only if:
 
-- The other module is bound to the "ibcaccount" port.
-- The channel being created is ordered.
-- The version string is "ics27-1".
+- The channel being created is ordered
+- The version string is "ics27-1"
```

Comment:
Is there a need to check if port id is equal to account address and that message is signed with the address owner, or we don't care if someone else create interchain account in name of some owner's address as only owner can control it after?

Reply:
It is a good point, as technically anyone can create a channel. I think it's okay to allow anyone to create a channel as the account can only be controlled by the owner anyway. 

I'm not sure how we would enforce this otherwise. The relayer will be signing these messages right?

Reply:
Since `ChanOpenInit` is being initiated by module from the `RegisterAddress` function, we can check address against the signer of the `RegisterAddress` message

Reply:
But this should be checked on controlling chain, NOT host chain as this will only get relayer-signed messages

Reply:
Since the code on controller chain is enforcing this property we don't need a check at all

Reply:
True. We do need a check on `executeTx` (host chain) to ensure that the signer of a message (returned from getSigners()) is in fact the interchain account address associated with the SourcePortId. Impl;cemented like so: https://github.com/cosmos/ibc-go/blob/interchain-accounts/modules/apps/ibc-account/keeper/relay.go#L144